index.bs

```diff
@@ -362,6 +365,22 @@ When <dfn>attempting to access the depth buffer</dfn> of {{XRDepthInformation}}
 
 </div>
 
+XRDepthView {#xr-depth-view_section}
+---------------------
+<script type="idl">
+[SecureContext, Exposed=Window]
+interface XRDepthView {
+  readonly attribute Float32Array projectionMatrix;
+  [SameObject] readonly attribute XRRigidTransform transform;
+};
+</script>
+
+An {{XRDepthView}} is simply a more restrictive version of an {{XRView}}. It contains an associated [=XRView/session=], [=XRView/frame=]. [=XRView/underlying view=], and [=XRView/internal projection matrix=].
```

Comment:
Nitpick: `An {{XRDepthView}} is simply a more restrictive version of an {{XRView}}.` reads like something non-normative.

Reply:
TBH, there's a bit of a mix here. The "It Contains" I think likely should be normative; but the statement that it's a more restrictive version of an XRView I agree does seem most likely to be non-normative. I can split into two lines.

Reply:
Obsolete; this has been removed.